draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -605,18 +812,18 @@ on bit strings that were not generated using the serialization procedure.
 
 ### Random Oracle {#term-rom}
 
-In practice, two types of mappings are common: (1) injective encodings, which
-can be used to construct a PRF as F(k, m) = k*H(m), and (2) random oracles,
+In practice, two types of encodings are common: (1) injective encodings, which
+can be used to construct a PRF as F(k, m) = k * H(msg), and (2) random oracles,
```

Comment:
Sure, that works. Or we can make it

    F(k, msg) = k * H(msg)

The reason I went through getting rid of `m` in this context is because the document defines `q = p^m` elsewhere and I was hoping to avoid overloading.

What do you think?

Reply:
Agreed -- let's go with your suggestion to avoid confusion!